app/assets/stylesheets/file.scss

```diff
@@ -101,9 +101,8 @@ body {
     vertical-align: top;
   }
 
-  /* Hide collapsed rows */
-  tr.hidden, tr.search-hidden {
-    display: none;
+  tr[hidden] { 
```

Comment:
I don't think this is needed because you never set the display style to anything else, right?

Reply:
It doesn't work without it.